src/sql/workbench/parts/notebook/notebookStyles.ts

```diff
@@ -21,6 +21,22 @@ export function registerNotebookThemes(overrideEditorThemeSetting: boolean, conf
 		let darkBoxShadow = '0px 4px 6px 0px rgba(0, 0, 0, 1)';
 		let addBorderToInactiveCodeCells = true;
 
+		// Book Navigation Buttons
+		const foregroundColor = theme.getColor(buttonForeground);
```

Comment:
minor: maybe call this `buttonForeground` in case we have other foreground values in the future. Same for `backgroundColor`

Reply:
changed to buttonForegroundColor / buttonBackgroundColor